### ISSUE TITLE
Изменить формулу начисления кураторских

### ISSUE THREAD

**Issue by yurgent71** [GolosChain/golos#928]
Предлагаю Изменить формулу начисления кураторских. 
25% - минимальный уровень. Скажем 75% - максимальный. 
Я бы оставил некоторый элемент геймификации в получении дополнительных кураторских, можно было бы немного модернизировать существующее правило, когда человек получает пропорционально доли своего апвота от общего количества проголосовавших до него (т.е. надо успеть апнуть до крупной рыбы). Также можно сделать дополнительные отчисления кураторам при больших выплатах за пост, но тут надо аккуратно подходить, чтобы не скатиться опять к голосованию только за топ-40 постов в сутки.

**Comment by yurgent71** [comment]
Можно общий (предварительный) % кураторских, получаемых за пост расчитывать по алгоритму:

k=1.0 // коэф.выставляется делегатами! (от 0,01 до 10)
n = 24 + log(общая выплата в голосах * k)
if n < 25 then n = 25
if n > 50 then n = 50

Индивидуальный процент кураторских по каждому высчитывать по следующему алгоритму:

первые 30% проголосовавших за пост получают по среднему проценту (посчитанному по алгоритму выше)

Остальные 70% проголосовавших получают в зависимости от % их апвота от общего количества на момент голосования, но не менее 25% и не более 75%. Можно также через log:

k1 = 1 /// коэф.выставляется делегатами! (от 0,01 до 1000)
n_ind=n + log(процент доли апвота куратора на момент голосования * k1)
if n_ind < n then n_ind = n
if  n_ind > 75 then n_ind =75

таким образом общий окончательный процент кураторских рассчитывать как средний процент от n_ind

Я особо не просчитывал все варианты, это скорее пример чтобы разжевать логику предложенного выше. 
также обращаю внимание, что коэф k и k1 вводятся для возможности делегатам влиять на формулу (при 0.01 - для всех будут 25%, а при очень больших цифрах - процент можно наоборот сместить к верхней границе)

**Comment by gropox** [comment]
Немного непонятно. Сейчас 25% авторских распределяются среди кураторов и 75% уходят автору.

Что предлагается? Распределять эти 25% по другому или распределять 75% авторского вознаграждения?

**Comment by yurgent71** [comment]
Сейчас кураторам уходит сумма менее 25% (из-за возможности резки кураторских). Предлагается сделать минимум 25, а максимум 75. Повышенный процент кураторских зависит от 2 факторов - общего сбора постом средств и времени голосования конкретного куратора относительно других (т.е. примерно как и сейчас считается)

**Comment by gropox** [comment]
> Повышенный процент кураторских зависит от 2 факторов - общего сбора постом средств и времени голосования конкретного куратора относительно других.

Вот тут и непонятно. То есть части авторов раздавать вознаграждения из 25% авторского вознаграждения, а отельным кураторам из дополнительных 50% 

А самому автору только 25%?

**Comment by yurgent71** [comment]
Да. Сам автор получит 25% если его пост соберет, скажем все голоса всех кураторов. В реальности же будет курс плавать от 25 до 30%, но у делегатов будут рычаги, влияющие на волатильность (т.е. 75% будет если все делегаты коэфициенты к и к1 задерут на максимальные значения).

**Comment by yurgent71** [comment]
Подчеркиваю, что предложенный алгоритм очень примерный. Скорее для понимая. В реальности надо будет еще встроить некоторую защиту, например при минимальных кол-вах голосов брать просто 25% (без всех этих сложностей) или же куратору-киту, у которого доля апвота более 90% брать по среднему. И т.п.

**Comment by afalaleev** [comment]
В результате в игру будут играть боты, которые будут просчитывать, где выгоднее поставить ап. Для человека, который ставит апы за то, что ему реально нравится, такие ухищрения в вычислениях только лишние.

**Comment by yurgent71** [comment]
Прикол в том, что боты не умеют предсказывать будущее. На малособирающих постах играть не получится - там будет 25%. А вот на интересных или прокачиваемых постах какая-то игра будет, но без возможности точно просчитать исход для играющего. Например, как угадать - попадаешь ли ты в 30% первых проголосовавших или нет, если не знаешь общее число. Более-менее просчитать можно будет только если подходит окно выплат, но и тут могут вмешаться другие игроки.
Короче, должно получиться весело - сложность в прогнозировании результата выплат приведет к изменению в моделях поведения бустероманов. Ну а если все это не понравится делегатам, то они могут просто вернуться к скучным 25%.

**Comment by yurgent71** [comment]
В любом случае это добавит игровой эффект. Боты и сейчас есть, так что картину не испортят они. Ну а в крайнем случае делегаты подкрутят коэффициенты
